conda/recipes/libcuml/meta.yaml

```diff
@@ -67,21 +85,29 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
-      build:
-        - cmake {{ cmake_version }}
```

Comment:
Looks like dropping `cmake` is causing [CI failures]( https://github.com/rapidsai/cuml/actions/runs/5512880413/jobs/10050335923?pr=5318#step:7:1561 )?

Guessing other packaging steps below may also need `cmake`

```
/opt/conda/conda-bld/work/install_libcuml.sh: line 6: cmake: command not found
```

Reply:
Yeah, I thought it was unnecessary so was trying to simplify the recipe... which seems like not a good idea :P